src/main/java/application/AbstractBaseController.java

```diff
@@ -157,6 +161,10 @@ private String getExceptionEmailBody(Exception exception){
         return "<html>" +
                 "<h3>Message</h3>" +
                 "<p>" + exception.getMessage() + "</p>" +
+                "<h3>Request URI</h3>" +
+                "<p>" + req.getRequestURI() + "<p>" +
```

Comment:
ok so double checked and those two methods don't return anything. i think this is because we send the information in the post body. when i try and read that information using `getInputStream` it throws an error saying that that stream has already been closed. i believe this is due to the jackson api library reading the post data when the request is first made